dockerfiles/gleam-1.9.Dockerfile

```
@@ -1,36 +1,24 @@
-# COPY --from=node_builder /app-cached /app ??? 
-
 ############################################################################## 
 # Erlang target
 FROM ghcr.io/gleam-lang/gleam:v1.9.1-erlang-alpine AS erlang_builder
 
-WORKDIR /build
-COPY --exclude=.git --exclude=README.md . /build
+ENV CODECRAFTERS_DEPENDENCY_FILE_PATHS="gleam.toml,manifest.toml"
+
+WORKDIR /app
+COPY --exclude=.git --exclude=README.md . /app
```

Hadolint (2.12.0)

[error] invalid flag: --exclude

(DL1000)